Runtime/UI/SafeArea.cs

```diff
@@ -0,0 +1,95 @@
+using UnityEngine;
+
+namespace Graphy.Runtime.UI
+{
+    /// <summary>
+    ///     Component that matches the size of the RectTransform to the safe area.
+    /// </summary>
+    [ExecuteAlways]
+    public sealed class SafeArea : MonoBehaviour
+    {
+        private Canvas m_canvas;
+        private bool m_isInitialized;
+        private Rect m_latestSafeArea;
+        private RectTransform m_rectTransform;
+        private DrivenRectTransformTracker m_rectTransformTracker;
+
+        private RectTransform RectTransform
+        {
+            get
+            {
+                if (m_rectTransform == null)
+                    m_rectTransform = (RectTransform)transform;
```

Comment:
I would extract this into Awake() so we make sure we only do it once. We control the entire UI setup, so we don't need to account for constant safety checks like this

Reply:
Changed to do so within `Awake()`.
In EditMode, `Awake()` is called only when a component is attached, so I changed rectTransform to SerializeField so that it is serialized when attached.